### PR TITLE
Update Docker image configuration

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request updates from Python `3.10.5` to `3.10.7`, updates dependencies, and switches from using Debian Bullseye to Alpine Linux 3.16.

## 💭 Motivation and context ##

This includes the following pull requests:
- #53
- #54
- #55
- #65
- #66
- #67

This fixes the currently failing build process due to no longer available dependency versions in addition to the improvements included in the above PRs.

**Note**: I have performed this testing for the latest (`0.1.0-rc.1`) pre-release as well.

## 🧪 Testing ##

Automated testing passes. Running the container locally I saw expected behavior.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍 👍

**Review by jsf9k — APPROVED** [review]
Nice cleanup!

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍 👍

File: Dockerfile [hunk 1]
```diff
@@ -1,64 +1,96 @@
...
-    chromium-common=102.0.5005.115-1~deb11u1 \
-    libxml2-dev=2.9.10+dfsg-6.7+deb11u2 \
-    libxslt1-dev=1.1.34-4 \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
+RUN apk --no-cache add \
+  ca-certificates=20220614-r4 \
+  chromium=110.0.5481.177-r0 \
+  libxml2=2.10.3-r1 \
+  libxslt=1.1.37-r1
```
Comment (jsf9k) [inline Dockerfile:75]: Good idea using the standard (vice `*-dev`) packages here!

**Review by jsf9k — APPROVED** [review]
_Approval intensifies!!!_

**Review by dav3r — APPROVED** [review]
Still 👍 👍

**Pull request merged**